modules/ROOT/pages/architecture-visualization.adoc

```diff
@@ -0,0 +1,44 @@
+= Architecture Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== About the Architecture Visualization
+
+The Architecture Visualization shows the topology of your application network. In this visualization, you can assign change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.
+
+== View Information about One Service in your Network
+
+When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
+Anypoint Visualizer displays information about the node in a card. This card is divided into the following tabs:
+
+* *Details*: Provides information about the node.
+* *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc[tags] from this section.
+
+== Filtering your View
+
+When you select a node, you can click *Select dependents and dependencies*, which selects all nodes related to the service you selected.
+If a single node is selected and does not have any dependents or dependencies, this option is hidden.
+
+You can hide nodes through what you select in the canvas.
+
+* To hide a single node, select it, and then click *Hide selected service*.
+* To hide multiple nodes, multiselect them, and then click *Hide selected services*.
+* To hide all of the services other than the selected nodes, you can click *Hide other services*. If you want to hide everything that is not related to a given dependency stack, select a node and then click *Select dependents and dependencies*, which selects all nodes related to the service which you selected. Then, click *Hide other services*.
```

Comment:
Is all of the text in the third bullet item about hiding all but selected nodes?
If so, break after first sentence.
If not, create a separate list item.
This list is a little tricky because it looks like it is supposed to list alternative single-step options, but the options are mini-tasks: Hide Single Node, Hide Multiple Nodes, Hide All But Selected Nodes....

Reply:
```suggestion
* To hide all of the services other than the selected nodes, you can click *Hide other services*. 
If you want to hide everything that is not related to a given dependency stack, select a node and then click *Select dependents and dependencies*, which selects all nodes related to the service which you selected. Then, click *Hide other services*.
```